packages/wonder-blocks-button/components/button.js

```diff
@@ -80,6 +80,11 @@ export type SharedProps = {|
      */
     href?: string,
 
+    /**
+     * A target destination window for a link to open in.
+     */
+    target?: string,
```

Comment:
We can change this to what #378 does later.

Reply:
Yep!